project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -217,6 +217,13 @@ function c4m_og_menu() {
     'access arguments' => [1, 2],
   );
 
+  $items['node/%node/merge'] = array(
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('c4m_og_merge_groups_build_form', 1),
+    'access arguments' => ['merge group content'],
+    'title' => 'Merge content',
```

Comment:
We should test for the node being a group.

Reply:
The check was done in the build form function.